Packs/IntegrationsAndIncidentsHealthCheck/Scripts/GetFailedTasks/GetFailedTasks.py

```diff
@@ -80,42 +43,31 @@ def get_failed_tasks_output(tasks: list, incident: dict):
     return task_outputs, number_of_error_entries
 
 
-def get_incident_data(incident: dict, tenant_name: str, rest_api_instance_to_use: str):
+def get_incident_data(incident: dict):
     """
         Returns the failing task objects of an incident.
 
         Args:
             incident (dict): An incident object.
-            tenant_name (str): The tenant of the incident.
-            rest_api_instance_to_use (str): A Demisto REST API instance name to use for fetching task details.
 
         Returns:
             tuple of context outputs and total amount of related error entries
     """
-    if tenant_name:
-        uri = f'acc_{tenant_name}/investigation/{str(incident["id"])}/workplan/tasks'
-    else:
-        uri = f'investigation/{str(incident["id"])}/workplan/tasks'
-
-    response = demisto.executeCommand(
```

Comment:
let's preserve this option for backwards compatibility, and use it if the `rest_api_instance` argument is provided explicitly (e.g. when we use this script to check an external XSOAR server)

Reply:
done